Dockerfile

```diff
@@ -7,4 +7,6 @@ from alpine:3.8
 expose 3333
 copy --from=builder /broker /broker
 add ./config.json /config.json
-cmd ["/broker","/config.json"]
+run mkdir -p /services
+copy services/* /services
```

Comment:
I think this can be done in one step with only the copy and without the "run"

Reply:
I vaguely remember trying only the `copy` and that didn't work.  worth giving it another try.